test/util/k8s/namespace.go

```diff
@@ -233,3 +233,9 @@ func GetMappingNamespaces(ctx context.Context, client TestClient, excludeNS []st
 	}
 	return joinedNsMapping, nil
 }
+
+func KubectlCreateNamespace(ctx context.Context, name string) error {
+	args := []string{"create", "namespace", name}
+	fmt.Println(args)
+	return exec.CommandContext(ctx, "kubectl", args...).Run()
+}
```

Comment:
Line 31 or so already have this. Do we need a new func?
```
func CreateNamespace(ctx context.Context, client TestClient, namespace string) error {
	ns := builder.ForNamespace(namespace).Result()
	// Add label to avoid PSA check.
	ns.Labels = map[string]string{
		"pod-security.kubernetes.io/enforce":         "baseline",
		"pod-security.kubernetes.io/enforce-version": "latest",
	}
	_, err := client.ClientGo.CoreV1().Namespaces().Create(ctx, ns, metav1.CreateOptions{})
	if apierrors.IsAlreadyExists(err) {
		return nil
	}
	return err
}
```

Reply:
Here I prefer to add a new function by using `kubectl` although I have no strong argument for this, in my opinion, using `kubectl` CLI for E2E test is needed somehow.